taxi/urls.py

```diff
@@ -16,7 +22,25 @@
         ManufacturerListView.as_view(),
         name="manufacturer-list",
     ),
+    path(
+        "manufacturers/create",
+        ManufacturerCreateView.as_view(),
+        name="manufacturer-create"
+    ),
+    path(
+        "manufacturers/<int:pk>/update",
+        ManufacturerUpdateView.as_view(),
+        name="manufacturer-update"
+    ),
+    path(
+        "manufacturers/<int:pk>/delete",
+        ManufacturerDeleteView.as_view(),
+        name="manufacturer-delete"
+    ),
     path("cars/", CarListView.as_view(), name="car-list"),
+    path("cars/create", CarCreateView.as_view(), name="car-create"),
+    path("cars/<int:pk>/update", CarUpdateView.as_view(), name="car-update"),
+    path("cars/<int:pk>/delete", CarDeleteView.as_view(), name="car-delete"),
```

Comment:
Consider adding a trailing slash to the 'cars/<int:pk>/delete' path for consistency with Django's URL conventions.